src/export/utils/walkArticle.ts

```diff
@@ -19,25 +19,25 @@ import { getImageSrc } from './getImageSrc';
 import { basekey } from './basekey';
 import { ISession } from '../../session/types';
 
-interface ArticleStateChild {
+export interface ArticleStateChild {
```

Comment:
@stevejpurves this was intentional to hide these. And force us to look through `ArticleState` as the source of truth. Not too fussed though.